src/Libraries/CoreNodeModels/DynamoConvert.cs

```diff
@@ -13,6 +13,7 @@ namespace CoreNodeModels
     [NodeName("Convert Between Units")]
     [NodeDescription("ConversionNodeDescription", typeof(Properties.Resources))]
     [NodeSearchTags("DynamoConvertSearchTags", typeof(Properties.Resources))]
+    [OutPortTypes("double")]
```

Comment:
Users don't understand what a double is. I think in the past we decided to use 'number' instead